### DISCUSSION TITLE
Campfire

### DISCUSSION THREAD

**Discussion by opekope2** [opekope2/StaffMod#32] [Staff blocks]
Campfire can be seen at [1:53-1:57](https://youtu.be/vlVUVsx6BYY?t=113) and [12:48-12:58](https://youtu.be/vlVUVsx6BYY?t=768) as a powerful Flammenwerfer

**Comment by opekope2** [comment]
I can't stop using ze Flammenwerfer staff, instead of developing the mod :joy:
It's almost ready

**Reply by charmander9695** [reply]
I guess its that fun
oh and just wondering, will it make you fly

**Reply by opekope2** [reply]
Yes, I already implemented it, just needs some adjustments
